TU20Bot/Models/CSVData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CsvHelper.Configuration.Attributes;
+
+namespace TU20Bot.Configuration {
+
+    public class CSVData {
```

Comment:
Name is too broad here. It sounds like this class is a general purpose for any and every csv file, but it isn't. Maybe `UserDetails`.